public/documents/fari-rpgs/charge-srd.md

```diff
@@ -372,6 +372,21 @@ Once you've decided how many charges you want to consume, roll and interpret the
 - On a `6`, the `consequence` is avoided.
 - On a `66`, the `consequence` is avoided, and you get +1 position for your next roll.
 
+> #### Designer Note: Hacking Determination Rolls
+> 
+> Determination rolls act as a mechanism for the players to say "no, I don't think so" when something bad happens to their character.
```

Comment:
Maybe use "resist" in intro since it's used below without much definition?

```suggestion
> Determination rolls act as a mechanism for the players to resist and say "no, I don't think so" when something bad happens to their character.
```